src/api/services/iuf/sessions.go

```diff
@@ -500,6 +501,19 @@ func (s iufService) getDagTasks(session iuf.Session, stageInfo iuf.Stage) ([]v1a
 	stage := stageInfo.Name
 	s.logger.Infof("create DAG for stage: %s", stage)
 
+	listTemplates := workflowtemplate.WorkflowTemplateListRequest{
+		Namespace: DEFAULT_NAMESPACE,
+	}
+	templates, err := s.workflowTemplateClient.ListWorkflowTemplates(context.TODO(), &listTemplates)
+	if err != nil {
+		return []v1alpha1.DAGTask{}, err
```

Comment:
Any reason we can't or should return `res` here which is defined above to be the same as this return value?

Also, a bit of an aside, but GoLand has a warning on line 500 that says, "Empty slice declaration using a literal," and it suggests replacing it with a nil slice declaration. I don't quite understand the distinction though, tbh.

Reply:
Good catch, will replace with res.

For line 500, linter just says it is better to do `var res []v1alpha1.DAGTask` than to do `res := []v1alpha1.DAGTask{}`. Maybe compiler does some optimizations in the former case.

Reply:
Done and unit tested.